src/main/java/hudson/plugins/chucknorris/RoundhouseAction.java

```diff
@@ -62,7 +62,7 @@
     /**
      * The run
      */
-    private transient Run<?, ?> mRun;
```

Comment:
It would actually cause persistency issues. Please do not do it unless strongly necessary